modules/electrophysiology_browser/jsx/components/electrophysiology_session_panels.js

```diff
@@ -0,0 +1,386 @@
+/**
+ * This file contains React component for Electrophysiology module.
+ *
+ * @author Alizée Wickenheiser.
+ * @version 0.0.1
+ *
+ */
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Panel from 'jsx/Panel';
+
+
+class FilePanel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: this.props.data,
+    };
+  }
+
+  /**
+   * Component did mount.
+   */
+  componentDidMount() {
+    console.log('componentDidUpdate check:');
+    console.log(JSON.stringify(this.state.data.downloads));
+    let downloads = [
+      'download_eeg_file',
+      'download_electrode_info',
+      'download_channels_info',
+      'download_events',
+      'download_all_files',
+      'download_fdt_file',
+    ];
+    for (let i=0; i<this.state.data.downloads.length; i++) {
+      console.log(this.state.data.downloads[i]);
```

Comment:
Leftover console.log?

Reply:
oooops. Will be removed in the next commit